docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx==1.5.2
-sphinxcontrib-napoleon==0.6.0
+sphinx==1.6.2
```

Comment:
This is less important - just the requirements for building the documentation. I think it is okay to be pinned.